default.yml

```diff
@@ -1,7 +1,8 @@
 inherit_from: .rubocop-performance.yml
 
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.6
+  # NewCops: enable
```

Comment:
Do we want to leave this commented out line?

Reply:
oops. I took it out.